app/Jobs/ApplicationDeploymentJob.php

```
@@ -2463,35 +2741,189 @@ private function add_build_env_variables_to_dockerfile()
+
+        if ($modified) {
+            // Write the modified Dockerfile back
+            $dockerfile_base64 = base64_encode($dockerfile->implode("\n"));
+            $this->execute_remote_command([
+                executeInDocker($this->deployment_uuid, "echo '{$dockerfile_base64}' | base64 -d | tee {$dockerfile_path} > /dev/null"),
+                'hidden' => true,
+            ]);
+
+            $this->application_deployment_queue->addLogEntry('Modified Dockerfile to use build secrets: '.$dockerfile->implode("\n"), hidden: true);
+        }
+    }
```

PHPMD (2.15.0)

The method modify_nixpacks_dockerfile_for_secrets is not named in camelCase. (undefined)

(CamelCaseMethodName)